back-end/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/assignment_submission_db
-spring.datasource.username=example_user
-spring.datasource.password=password123
+spring.datasource.username=root
+spring.datasource.password=Setiawan112!
```

Comment:
Never check in your passwords. This will be a publicly available repository for the world to see.